packages/cli/src/ui/AppContainer.tsx

```diff
@@ -1011,6 +1011,26 @@ Logging in with Google... Please restart Gemini CLI to continue.
     showPrivacyNotice ||
     !!proQuotaRequest;
 
+  const needsBell = useRef(false);
+  useEffect(() => {
+    switch (streamingState) {
+      case StreamingState.Idle:
+      case StreamingState.WaitingForConfirmation:
+        if (needsBell.current) {
+          if (config.getEnableTerminalBell()) {
```

Comment:
also can you read this option of settings instead of config?

Reply:
Sorry not sure I understand?